File: app/src/main/java/wyyu/multi/cell_integer/data/DataIntegerA.java
package wyyu.multi.cell_integer.data;

import wyyu.multi.app.Constants;

/**
 * Created by wyyu on 2020-05-07.
 **/

public class DataIntegerA implements IIntegerData {

    public int index;

    public int resId;

    public String content;

    public DataIntegerA(int index) {
        this.index = index;
        this.resId = Constants.RES_ID;
        this.content = Constants.CONTENT;
    }

    @Override public int loadDataType() {
        return IIntegerData.DATA_A;
    }
}

File: app/src/main/java/com/wyyu/multiktest/cell_Integer/data/DataIntegerA.kt
package com.wyyu.multiktest.cell_Integer.data

import com.wyyu.multiktest.app.Constants

class DataIntegerA(
    val index: Int,
    val resId: Int = Constants.RES_ID,
    val content: String = Constants.CONTENT
) : IDataInteger {

    override fun dataType(): Int {
        return IDataInteger.DATA_A
    }
}